### ISSUE TITLE
Allow for creating a Jira ticket from Quality-time

### ISSUE THREAD

**Issue by Sebastiaan127001** [ICTU/quality-time#2931]
# User story

As a quality manager I want to create an issue in the configured issue tracker and have it automatically linked from Quality-time so that I can create and link issues in one step.

## Acceptance criteria

The story is considered complete if:
- [x] The user can create an issue for a metric in the configured issue tracker from Quality-time, and
- [x] The issue is linked from the metric, and
- [x] The metric is linked from the issue

## Assumptions 

- An issue tracker is configured.
- The configured issue tracker user has the necessary rights to create an issue in the issue tracker.
- If the Quality-time user is a different user than is configured for the issue tracker, the Quality-time user can still edit the newly created issue in the issue tracker.

## Out of scope

- Quality-time doesn't decide on its own to create tickets.
- Automatically accept the metric as technical debt (if red or yellow)
- Automatically getting the available project keys and issue types from Jira.

# Tasks

### Frontend

- [x] Add a field for a default project for new issues to the issue tracker configuration
- [x] Add a field for a default issue type for new issues to the issue tracker configuration
- [x] Add a button next to the issue tracker id field "Create new issue for this metric"
- [x] Button should be inactive if there's no issue tracker configured.
  - [x] Button should be inactive if the user removes the issue tracker. 
- [x] Clicking the button invokes the external server endpoint for creating issues. 
- [x] The frontend uses the issue id returned by the external server to open a new browser tab for the user to change the issue with extra information.
- [x] If the configured issue tracker user has no rights to create issues in the issue tracker or something else goes wrong, show an appropriate error message using the toaster.

### External server

- [x] Add endpoint to create an issue for a specific metric. The endpoint uses the issue tracker (e.g. Jira) API to create a new issue (with the metric name as title), inserts the issue id into the issue id field, and returns the issue id to the frontend.

## Feature tests

- [x] User tries to create a new issue (really creating an issue would require a fake issue tracker)

## Documentation

- [x] Update changelog
- [x] Add instruction to user manual
  - [x] Mention that new tab may be blocked by the browser and user needs to allow Quality-time to open a new tab. 
- [x] Update screenshots

**Comment by Sebastiaan127001** [comment]
When there's no issue in Jira from a certain metric, you can instantly create it.
